webscrambles/src/net/gnehzr/tnoodle/server/webscrambles/ScrambleRequest.java

```diff
@@ -1312,12 +1338,23 @@ public static ByteArrayOutputStream requestsToZip(ServletContext context, String
         ZipOutputStream zipOut = new ZipOutputStream(baosZip);
         HashMap<String, Boolean> seenTitles = new HashMap<String, Boolean>();
 
+        // Computer display zip
```

Comment:
Could you add a quick comment here about the goals of this zip file? I think it's easy to get confused here about if zip files are created just to easily hold files vs to encrypt stuff. I'm thinking something like:

> This zip file is intentionally *not* password protected, it's just an easy way to distribute a collection of files. Any files we put in this zip must be encrypted themselves.

(feel free to reword, but I hope the gist of what I'm trying to say makes sense)

Reply:
Done, thanks!